src/components/send/SendHeader.js

```diff
@@ -87,12 +86,12 @@ export default function SendHeader({
   userAccounts,
   watchedAccounts,
 }) {
-  const profilesEnabled = useExperimentalFlag(PROFILES);
   const { setClipboard } = useClipboard();
   const { isSmallPhone, isTinyPhone } = useDimensions();
   const { navigate } = useNavigation();
   const { colors } = useTheme();
   const [hexAddress, setHexAddress] = useState('');
+  useRainbowProfile(hexAddress);
```

Comment:
hm so we don't use the info being calculated from this hook?

Reply:
nope, just using it as a prefetch. will prob add a separate prefetch method